src/server/api/area/index.ts

```diff
@@ -4,11 +4,13 @@ import { ApiEndPoint } from '@meta/api/endpoint'
 
 import { AuthMiddleware } from '@server/middleware/auth'
 
+import { getAreas } from './getAreas'
 import { postCountry } from './postCountry'
 
 export const AreaApi = {
   init: (express: Express): void => {
     // Country
     express.post(ApiEndPoint.Area.country(), AuthMiddleware.requireEditTableData, postCountry)
+    express.get(ApiEndPoint.Area.areas(), getAreas)
```

Comment:
Auth Middleware

Reply:
getCountries is used by the Landing page too

this means that anonymous user are able to get the country list